services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -190,9 +200,75 @@ specCreateUser = describe "POST /Users" $ do
   it "writes all the stuff to all the places" $
     pendingWith "factor this out of the PUT tests we already wrote."
 
--- | Test that a user can be created via SCIM and that it also appears on the Brig side.
-testCreateUser :: TestSpar ()
-testCreateUser = do
+testCreateUserWithPass :: TestSpar ()
+testCreateUserWithPass = do
+  env <- ask
+  tok <- do
+    (_, tid) <- call $ createUserWithTeam (env ^. teBrig) (env ^. teGalley)
+    registerScimToken tid Nothing
+  user <- randomScimUser <&> \u -> u {Scim.User.password = Just "geheim"}
+  createUser_ (Just tok) user (env ^. teSpar) !!! do
+    const 400 === statusCode
+    -- TODO: write a FAQ entry in wire-docs, reference it in the error description.
+    -- TODO: yes, we should just test for error labels consistently, i know...
+    const (Just "Setting user passwords is not supported for security reasons.") =~= responseBody
+
+testCreateUserNoIdP :: TestSpar ()
```

Comment:
The description of this test says it checks tht the email validation link is sent; but i dont see this being tested here.
How are we making sure this codepath is indeed taken?

Reply:
Scrap that ;I can't read :)